src/explorer/Main.hs

```diff
@@ -43,19 +46,28 @@ import           Pos.Util.UserSecret        (usVss)
 import           Pos.Util.Util              (powerLift)
 import           Pos.WorkMode               (RealMode, WorkMode)
 
+import           Pos.Explorer               (ExplorerBListener,
+                                             runExplorerBListener)
 import           Pos.Explorer.Socket        (NotifierSettings (..))
 import           Pos.Explorer.Web           (explorerPlugin, notifierPlugin)
 
 import           ExplorerOptions            (Args (..), getExplorerOptions)
 import           Params                     (getBaseParams, getKademliaParams,
                                              getNodeParams, gtSscParams)
 
+
+type ExplorerProd = ExplorerBListener (RealMode SscGodTossing)
+
+-- | Lift monadic computation to 'ExplorerBListener'.
+liftBListenerRedirects :: m a -> ExplorerBListener m a
+liftBListenerRedirects = coerce
```

Comment:
@akegalj I didn't know what `coerce` was myself (I saw the type signatures and trusted that), but I found that it's similar to PS wrap/unwrap of **newtypes** - https://wiki.haskell.org/GHC/Coercible.